src/Language/Fixpoint/Types/Names.hs

```diff
@@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
   , dropSym
   , headSym
   , lengthSym
+  , mappendSym
```

Comment:
Yes, I am creating the symbol from the refined annotation `instance ToProve a (Proof -> b) where`. Let me know if you find a better way:)